src/applications/financial-status-report/containers/IntroductionPage.jsx

```diff
@@ -21,9 +21,6 @@ const IntroductionPage = ({
   }, []);
 
   const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
-  const showLegacyWizard = useToggleValue(
-    TOGGLE_NAMES.showFinancialStatusReportWizard,
-  );
   const showStaticWizard = useToggleValue(TOGGLE_NAMES.fsrWizard);
```

Comment:
I think we can drop this feature flag as well! I was comparing what's live to [the last implementation ticket](department-of-veterans-affairs/va.gov-team/issues/106937), and it looks like the `fsr_wizard` feature is also fully enabled on production. I don't anticipate we'll need to roll back any of that functionality at this point, so we can safely remove the flag and its related content!